contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2451,15 +2453,32 @@ antlr_parser_cpp(const char *sourceText)
 		std::cout << sep << std::endl;
 	}
 
+	result = antlr_parse_query(sourceText, !pltsql_disable_sll_parse_mode);
+
+	// If parsing failed in SLL mode, reparse in LL mode
+	if (!result.success && result.errcod == ERRCODE_SYNTAX_ERROR)
```

Comment:
if pltsql_disable_sll_parse_mode == true
we don't need to go into this if block